packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -493,261 +489,303 @@ export class StatsAnalyzer extends EventsScope {
    */
   private compareLastStatsResult() {
     if (this.lastStatsResults !== null && this.meetingMediaStatus) {
-      // compare audio stats sent
-      let currentStats = null;
-      let previousStats = null;
-
-      Object.keys(this.statsResults).forEach((mediaType) => {
-        if (!this.lastStatsResults[mediaType]) {
-          return;
-        }
-
-        if (this.meetingMediaStatus.expected.sendAudio && mediaType.includes('audio-send')) {
-          currentStats = this.statsResults[mediaType].send;
-          previousStats = this.lastStatsResults[mediaType].send;
-
+      const getCurrentStatsTotals = (keyPrefix: string, value: string): number =>
+        Object.keys(this.statsResults)
+          .filter((keys) => keys.includes(keyPrefix))
```

Comment:
should we use startsWith() instead of includes() here? and also the variable name should be `key` not `keys`, right?

Reply:
Yes good call, updated.